### PR TITLE
Added handling for unsupported or invalid images

### PR DESCRIPTION
**Pull request by martijnvdwal**

### CODE REVIEW

File: Sources/Parley/Views/ParleyComposeView/ParleyComposeView.swift [hunk 1]
```diff
@@ -360,29 +360,25 @@ extension ParleyComposeView: KeyboardAccessoryViewDelegate {
...
+        let options = PHImageRequestOptions()
+        options.isNetworkAccessAllowed = true
+            
+        PHImageManager.default().requestImageData(for: asset, options: options) { [weak self] (data, _, _, _) in
+            if let data, let image = UIImage(data: data) {
+                picker.dismiss(animated: true, completion: {
+                    self?.delegate?.send(image: image, with: data, url: imageURL)
+                })
+            } else {
+                self?.delegate?.failedToSelectImage()
```
Comment (alexkok) [inline Sources/Parley/Views/ParleyComposeView/ParleyComposeView.swift:379]: Should we also do
```
            picker.dismiss(animated: true, completion: nil)
```
here?

Comment (martijnvdwal) [reply Sources/Parley/Views/ParleyComposeView/ParleyComposeView.swift:379]: Yeah I think you’re right, I will add that in.

**Review by alexkok — APPROVED** [review]

**Comment by alexkok** [comment]
Fixes #56

**Comment by alexkok** [comment]
Released in 3.9.1 🚀

**Pull request merged**